### PR TITLE
Xcode 7.3

### PR DESCRIPTION
**Pull request by danielaRiesgo**
## Sumary ##
**Trello card**: https://trello.com/c/UfltcTGZ/26-migrar-a-xcode-7-3

Updating swift syntax to new version.

### CODE REVIEW

**Comment by guidomb** [comment]
:+1:

**Comment by woloxci** [comment]
:+1: There are no linter violations.

File: Authentication/Login/View/LoginController.swift [hunk 1]
```diff
@@ -69,7 +69,7 @@ public final class LoginController: UIViewController {
         loginView.render()
         bindViewModel()
         navigationController?.navigationBarHidden = true
-        let tapRecognizer: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: "dismissKeyboard:")
+        let tapRecognizer: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(LoginController.dismissKeyboard(_:)))
```
Comment (FranDepascuali) [inline Authentication/Login/View/LoginController.swift]: I think you could write it as 
```swift
let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(dismissKeyboard))
```

**Comment by FranDepascuali** [comment]
👍

**Pull request merged**